src/main/java/frc/robot/RobotContainer.java

```diff
@@ -115,8 +116,9 @@ public void burnFlashSparks() {
    * joysticks}.
    */
   private void configureBindings() {
+    OuttakeSequence o = new OuttakeSequence(lift);
```

Comment:
Is this needed?

Reply:
No, sorry. This was left in from when we were trying to figure out why OuttakeSequence() wasn't working.